### PR TITLE
Interactive viewers with chemiscope 0.7

### PR DESCRIPTION
**Pull request by ceriottm**
Putting the new chemiscope release to good use. This was started as a testing ground for the chemiscope 0.7 development, now that it works I converted all examples that used chemiscope to actually visualize.

### CODE REVIEW

File: developer/get_examples.py
Comment (Luthaf) [inline developer/get_examples.py]: why is this required?

File: examples/README.rst
Comment (Luthaf) [inline examples/README.rst:1]: this should not be needed

File: docs/src/conf.py
Comment (Luthaf) [inline docs/src/conf.py]: this should not be `gen_gallery`. We generate the galleries in a sperate step, this final sphinx build should just load pre-generated rst

File: docs/src/conf.py
Comment (Luthaf) [inline docs/src/conf.py]: this is not needed, the configuration for rst generation is in the root `generate-gallery.py`

File: docs/requirements.txt
Comment (Luthaf) [inline docs/requirements.txt]: I don't think we need matplotlib here. chemiscope is fine, since we are using the corresponding sphinx extension

File: developer/get_examples.py
Comment (sofiia-chorna) [reply developer/get_examples.py]: I thought that `nox -e docs` treats all folders as the instance of the example to generate the docs.
However, with the new development, there are some generated datasets that are stored in the folder `_chemiscope_sphinx_data`.
Hence I thought to exclude this generated folder out to prevent counting it as an "example".

Comment (Luthaf) [reply developer/get_examples.py]: what's generating data there? Ideally anything generated should be stored in a sub-directory of the main example directoty (i.e. `roy-gch/_chemiscope_sphinx_data`), and then we can gitignore this sub-directory

**Comment by sofiia-chorna** [comment]
As the `docs/build/html` is usually deployed to the website, for testing it is necessary to run the server from this folder.
Also, I am aware of the issue with the not found json, it is related to the fact of having to configurations, but I am working on adapting the code in the chemiscope to handle it without error...

**Comment by sofiia-chorna** [comment]
Should be OK right now

**Comment by ceriottm** [comment]
Just to say that this seems to work well for me but I won't have time to contribute any polishing until Tue at the earliest. Anyone else is welcome to chime in. I think we should, in any case, first merge the chemiscope PR and make a release, so it does not take 2h to make the docs from scratch because of the time it takes to compile CS from source.

**Comment by DavideTisi** [comment]
Not sure what i am suppose to see, if i download the documentation and open it, everything is the same as usual.
I still get the usual message `Interactive chemiscope widgets can only be visualized inside a jupyter notebook.`

**Comment by ceriottm** [comment]
> Not sure what i am suppose to see, if i download the documentation and open it, everything is the same as usual. I still get the usual message `Interactive chemiscope widgets can only be visualized inside a jupyter notebook.`

roy-GCH should have a chemiscope, but you need to open it with with a http server

**Comment by sofiia-chorna** [comment]
@DavideTisi Can you please run http.server from the docs/build/html? 
```
cd docs/build/html
python3 -m http.server
```

**Comment by DavideTisi** [comment]
but i should clone the repo i cannot only download [documentation.zip](https://nightly.link/lab-cosmo/software-cookbook/actions/artifacts/1574509352.zip) ?

**Comment by ceriottm** [comment]
you are right `/_chemiscope_sphinx_data/` does not get bundled into the zip. IDK what it means @sofiia-chorna

File: examples/gaas-map/environment.yml
Comment (Luthaf) [inline examples/gaas-map/environment.yml]: as mentioned in the main discussion, let's not merge this until we have a new chemiscope release.

**Review by Luthaf — CHANGES_REQUESTED** [review]

**Comment by sofiia-chorna** [comment]
The workflow of github-actions includes the files to `documentation.zip` from the `docs/src/examples/*`, but the folder is not under this location, I will fix it from the code of chemiscope to not change the current workflow

**Comment by sofiia-chorna** [comment]
It should be OK with the `documentation.zip` now. To launch, it is necessary to execute `python3 -m http.server` at the root of the unzipped documentation file.

`documentation.zip` is here: https://github.com/lab-cosmo/software-cookbook/actions/runs/9447974424

**Comment by DavideTisi** [comment]
it works form me very good once chemiscope release is out we can merge

**Comment by ceriottm** [comment]
@DavideTisi this looks fine on my side. Can you test it works so we can close it and announce BOTH cs 0.7 and the software cookbook?

**Review by DavideTisi — APPROVED** [review]

**Pull request merged**
